### ISSUE TITLE
[Suggestion] Truth or dare with Monika

### ISSUE THREAD

**Issue by VirginiaRocks** [Monika-After-Story/MonikaModDev#4244]
Hello! I'm back with another suggestions :) 

1. Why not add a game of truth or dare with Monika? I thought about this 10 minutes ago and I wanted to share it before forgetting about it... 
This might sound strange and surely a little difficult, but I thought about it in a certain way...

(Example)
Monika: Truth or dare [Player]?
[Choice between Truth or dare]

Dare:
Monika: Mh...let me think...
Monika: Oh, I got it!
Monika: I dare you to say that you love me! Ehehe~
Or:
Monika: I dare you to play a game of pong with me! 

And things like that... I'm currently thinking more things to say.

Truth: 
Monika: Mh... let me think...
Monika: Oh, I got it!
Monika: Do you love me?
[The answer should only be yes]
Or:
Monika: Do you like this mod?
[Just for breaking the fourth wall more]
Or:
Monika asks you something about her that she said ("Am I vegetarian?" etc.)

When you ask Monika Truth or dare, she chooses. 
Truth:
[There are some options of the "ask" menu or some questions that I didn't imagine yet. You click on one and Monika answers]

Dare:
[There are dares like: "Kiss me" (If you already received the first kiss], "Make a ... face" (angry, sad, strange, happy...) "Scare me randomly " (with a jumpscare that Monika could do in a random moment while playing) "Change your clothes in..." (when more clothes will be added) etc.]

I'm still upgrading this idea because for now is just a random thought.
When you finish the things to say and you still want to play truth or dare Monika could say:

Monika: I'm sorry [Player], but I'm running out of things to say... do you want to play anyway?
[Choice between Yes or No]

[Yes]
Monika: Okay then! Who begins?
[No]
Monika: Then, what do you want to play?

Like I said, it's more a work in progress... if someone has a suggestion I would love to hear it!
Thank you for reading!

**Comment by Thimeo** [comment]
I think this could have the potential to be a fun game to play with Monika, but I see 2 problems in it.

1) There is a long list of very simple game that have been suggested already and that could be played with Monika (@Rai99 could tell you more about it), and so far option like solitaire or battleship are in the progress of making. These are very simple games to implement and have a lifespan almost infinite.

2) For this game to be effective (I mean by that, that there must be a number of different outcome for each choice and each player) it would require A LOT of dialogue, and as #3784 confirms, we really have a shortage of those. So such a dialogue-massive game would be very difficult to implement, although not impossible if the right peoples writes it.

In conclusion (and I precise that I'm neither a mod or even someone that implicated in the project) I think that it could be good to keep it as a possible interesting idea, but this category already have so many things in here that it's difficult for any of them to become real.

**Comment by Rai99** [comment]
I'm not sure if this kind of game is realistically possible with a reasonable amount of depth. Playing this game in real life, you could ask or dare anything possible, which is probably what makes the game interesting. 

For Monika, I don't think there's really much we can do apart from just having a list of dares. Monika won't even be able to tell if you do them or not. As for dares for Monika, I feel like there's almost nothing we can do. Daring to change clothes is a possible idea, although I don't think it's a very satisfying dare, not to mention new clothes are extremely costly to implement, and probably not worth it just for a game of truth or dare. Sure, you could ask Monika to make a face or kiss you, but what else? I've never played truth or dare myself, but those seem really, really tame compared to what I've heard about the game. Kissing might be extreme, but would be normal for Monika since it probably wouldn't be your first kiss. The first kiss isn't something I think we hand out easily. 

Same goes for truth pretty much. There's virtually no room for meaningful questions worth asking in a game of truth or dare with Monika, especially considering how close Monika likely is to you, so she would probably answer you honestly in any situation already. 

Truth or dare is more of a group game I think, and especially doesn't work if you're really close to the people involved.

**Comment by VirginiaRocks** [comment]
> I'm not sure if this kind of game is realistically possible with a reasonable amount of depth. Playing this game in real life, you could ask or dare anything possible, which is probably what makes the game interesting.
> 
> For Monika, I don't think there's really much we can do apart from just having a list of dares. Monika won't even be able to tell if you do them or not. As for dares for Monika, I feel like there's almost nothing we can do. Daring to change clothes is a possible idea, although I don't think it's a very satisfying dare, not to mention new clothes are extremely costly to implement, and probably not worth it just for a game of truth or dare. Sure, you could ask Monika to make a face or kiss you, but what else? I've never played truth or dare myself, but those seem really, really tame compared to what I've heard about the game. Kissing might be extreme, but would be normal for Monika since it probably wouldn't be your first kiss. The first kiss isn't something I think we hand out easily.
> 
> Same goes for truth pretty much. There's virtually no room for meaningful questions worth asking in a game of truth or dare with Monika, especially considering how close Monika likely is to you, so she would probably answer you honestly in any situation already.
> 
> Truth or dare is more of a group game I think, and especially doesn't work if you're really close to the people involved.

This is why I already imagined that the idea probably wouldn't fit in the game 😅 I wanted to propose it anyway.
Thank you for answering!

**Comment by ThePotatoGuy** [comment]
this kind of game is too open-ended. [rejecting]
